resource/conf.go

```diff
@@ -8,8 +8,35 @@ type Conf struct {
 	// no default page size is set resulting in no pagination if no `limit` parameter
 	// is provided.
 	PaginationDefaultLimit int
+	// ForceTotal controls how total number of items on list request is computed.
+	// By default (User), if the total cannot be computed by the storage andler for
+	// free, no total metadata is returned until the user explicitely request it
+	// using the total=1 query-string parameter. Note that if the storage cannot
+	// compute the total and does not implement the resource.Counter interface,
+	// a "not implemented" error is returned.
+	//
+	// The Always mode always force the computation of the total (make sure the
+	// storage either compute the total on Find or implement the resource.Counter
+	// interface.
+	//
+	// Denied prevents the user from requesting the total.
+	ForceTotal ForceTotalMode
 }
 
+// ForceTotalMode defines Conf.ForceTotal modes
```

Comment:
Adding a trailing period, also for single-sentence comments, would be [more Idiomatic](https://golang.org/doc/effective_go.html#commentary), and in line with how the standard library [do single-sentence comments](https://golang.org/pkg/io/#ByteWriter). This is a small detail, so feel free to ignore it if you prefer this style.


The concrete text I am referring to from Effective Go is:

 > Doc comments work best as complete sentences, which allow a wide variety of automated presentations. The first sentence should be a one-sentence summary that starts with the name being declared.

P.S! Last time I did a PR for rest-layer, I chose not to add the trailing period to stay consistent with other comments in the same file.